src/checkout/checkout-selector.ts

```diff
@@ -83,20 +82,7 @@ export function createCheckoutSelectorFactory(): CheckoutSelectorFactory {
             const grandTotal = checkout.grandTotal || 0;
             const storeCredit = checkout.customer.storeCredit || 0;
 
-            return useStoreCredit ? Math.max(grandTotal - storeCredit, 0) : grandTotal;
-        }
-    );
-
-    const getOutstandingBalance = createSelector(
-        getCheckout,
-        getCheckout => () => {
-            const checkout = getCheckout();
-
-            if (!checkout) {
-                return;
-            }
-
-            return checkout.outstandingBalance;
+            return useStoreCredit ? Math.max(grandTotal - storeCredit, 0) : checkout.outstandingBalance;
```

Comment:
Can you make it so that if `useStoreCredit` is not passed (`undefined`), return `checkout.outstandingBalance`?